src/transformers/training_args.py

```diff
@@ -407,21 +407,28 @@ class TrainingArguments:
               `"shard_grad_op"`).
             - `"auto_wrap"`: Automatically recursively wrap layers with FSDP using `default_auto_wrap_policy`.
         fsdp_min_num_params (`int`, *optional*, defaults to `0`):
```

Comment:
You can remove the documentation of the deprecated argument, but make sure that when the user sets it, we issue a deprecation warning and update the fsdp_config (in the post init).